tests/unit/evaluators/ScatterScalarResponse.cpp

```diff
@@ -74,31 +74,31 @@ TEUCHOS_UNIT_TEST(evaluator_unit_tester, separableScatterScalarResponseHessianVe
     auto overlapped_p_space = p_dof_mgr->ov_indexer()->getVectorSpace();
     auto overlapped_x_space = x_dof_mgr->ov_indexer()->getVectorSpace();
 
-    const auto hess_vec_prod_g_xx = Thyra::createMember(x_space);
```

Comment:
I don't follow these changes. Why do we need MV's (with 1 col)?

Reply:
I'm not 100% sure what the cause is but these unit tests started failing when I tried to grab the HessianWorkset multivector view using `getNonconstDeviceData`. I was getting an exception that the Thyra_Multivector couldn't be cast to a Tpetra one. Looking at it in a debugger made me think something went wrong when casting a Thyra_Vector to a Thyra_Multivector. Changing the unit tests to use MVs from the start fixed it.

Reply:
Ah, these are unit tests. Then it's ok.